test/e2e/serial/tests/configuration.go

```diff
@@ -1510,3 +1523,19 @@ func getLabelRoleWorker() string {
 func getLabelRoleMCPTest() string {
 	return fmt.Sprintf("%s/%s", depnodes.LabelRole, roleMCPTest)
 }
+
+func isCustomPolicySupportEnabled(nro *nropv1.NUMAResourcesOperator) bool {
+	GinkgoHelper()
```

Comment:
so here we do want to call `GinkgoHelper`?

Reply:
the main purpose of GinkgoHelper() is to avoid using offset() for location calculation of the running code. It is especially helpful to be called on functions that are highly reused or are called several times in one spec. In that case it would be more clear to debug failures to get the location of the failing call rather than the failing line within the helper function. 
Considering this helper function is short and well defined I didn't mind to add the GinkgoHelper() call here. If this causes confusion we can remove it and address all that whenever the function is highly requested, then we can propose a cleanup PR to add this call if needed.

Reply:
I'm not completely sure I follow the logic, but this specific application makes sense to me, so let's move on

Reply:
thing is: the problem I'm seeing (and which prompted the previous conversation?) was related to bad helpers which had too many expectations inside them and whose failure was hard to debug. This is still kinda related to the helper length, but the key discriminatror should be number and likeness of failure of expectations inside the helper rather than the helper length itself. Anyway, overall makes sense to me so we can discuss nuances later on.